driver-core/src/main/com/mongodb/internal/connection/ProtocolExecutor.java

```diff
@@ -20,9 +20,13 @@
 import com.mongodb.internal.session.SessionContext;
 
 public interface ProtocolExecutor {
-    <T> T execute(LegacyProtocol<T> protocol, InternalConnection connection);
+    default <T> T execute(LegacyProtocol<T> protocol, InternalConnection connection) {
```

Comment:
Made these default methods that are unsupported, since LB mode doesn't use legacy protocol.